var/spack/repos/builtin/packages/libpressio-sperr/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class LibpressioSperr(CMakePackage):
+    """A LibPressio plugin for Sperr"""
+
+    homepage = "https://github.com/robertu94/libpressio-sperr"
+    url = "https://github.com/robertu94/libpressio-sperr/archive/refs/tags/0.0.1.tar.gz"
+    git = homepage
```

Comment:
(the git url is broken)

Reply:
Also works for me?

```console
[runderwood@defiant tmp]$ git clone "https://github.com/robertu94/libpressio-sperr"
Cloning into 'libpressio-sperr'...
remote: Enumerating objects: 22, done.
remote: Counting objects: 100% (22/22), done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 22 (delta 6), reused 20 (delta 4), pack-reused 0
Receiving objects: 100% (22/22), 16.63 KiB | 1.28 MiB/s, done.
Resolving deltas: 100% (6/6), done.
[runderwood@defiant tmp]$ curl -LO "https://github.com/robertu94/libpressio-sperr/archive/refs/tags/0.0.1.tar.gz"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 15186    0 15186    0     0  52876      0 --:--:-- --:--:-- --:--:-- 52876

```

Reply:
My mistake: I misread this as setting `git = url` rather than `git = homepage`